### PR TITLE
CF-1180 - Tiamat error if preferences archive dc doesn't exist.

### PR DESCRIPTION
**Pull request by usnavi**
Updated tiamat to first query that the to-be-archived region exists in the liveUrlMap.  If it doesn't, ignores the region and doesn't create an archive

### CODE REVIEW

File: tiamat/src/main/scala/com/rackspace/feeds/archives/Archiver.scala [hunk 1]
```diff
@@ -305,10 +301,15 @@ class ArchiverHelper( prefMap : Map[String, TenantPrefs],
...
-          createFeed(container, key, content, impMap(tid), getFeedId, feedUuid, liveFeed)
-          None
+          // if live URL map doesn't contain the region, we don't have that region in the environment
+          liveUriMap.get(key.region) match {
 
+            case Some(liveFeed) =>
+              containerCheck (container, impMap (tid) )
+              createFeed (container, key, content, impMap (tid), getFeedId, feedUuid, liveFeed)
+              None
+            case None => None
```
Comment (shintasmith) [inline tiamat/src/main/scala/com/rackspace/feeds/archives/Archiver.scala]: Do we at least log a WARNING somewhere? If people are sending bad data, we might want to know, perhaps?

**Comment by ChandraAddala** [comment]
Looks good to me.

**Comment by usnavi** [comment]
@shintasmith  - you've been WARNED!

**Comment by shintasmith** [comment]
lgtm

**Pull request merged**